templates/siteinspector.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>siteinspector</Name>
+  <Repository>siteinspector/siteinspector</Repository>
+  <Registry>https://hub.docker.com/r/siteinspector/siteinspector</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/siteinspector/siteinspector/issues</Support>
+  <Project>https://github.com/siteinspector/siteinspector</Project>
+  <WebUI>http://[IP]:[PORT:808]</WebUI>
+  <Requires>Postgres &#xD;
+  Redis &#xD;
+  </Requires>
+  <PostParams>bash -c "rake db:migrate && foreman start"</PostParams>
+  <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/siteinspector.xml</TemplateURL>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/siteinspector.png</Icon>
+  <Overview>A tool for catching spelling errors, grammatical errors, broken links, and other errors on websites.</Overview>
+  <Config Name="Config" Target="/app/" Default="/mnt/user/appdata/siteinspector" Mode="rw" Description="Config Location" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/siteinspector</Config>
+  <Config Name="Web UI" Target="808" Default="808" Mode="" Description="Web UI Port" Type="Port" Display="always" Required="true" Mask="false">808</Config>
+  <Config Name="Secret Key Base" Target="SECRET_KEY_BASE" Default="" Mode="" Description="Secret Key Base" Type="Variable" Display="advanced" Required="false" Mask="false"></Config>
+  <Config Name="Enable Logging" Target="RAILS_LOG_TO_STDOUT" Default="true" Mode="" Description="Enable log reporting" Type="Variable" Display="always" Required="true" Mask="false">true</Config>
+  <Config Name="Database Connection" Target="DATABASE_URL" Default="postgresql://postgres:postgres@postgres:5432/siteinspector" Mode="" Description="Postgres Database Connection String" Type="Variable" Display="always" Required="true" Mask="false">postgresql://postgres:postgres@postgres:5432/siteinspector</Config>
+  <Config Name="Redis Connection" Target="REDIS_URL" Default="redis://redis:6379/0" Mode="" Description="Redis Connection String" Type="Variable" Display="always" Required="true" Mask="false">redis://redis:6379/0</Config>
+  <Config Name="Port for Web UI" Target="PORT" Default="808" Mode="" Description="Enable log reporting" Type="Port" Display="advanced" Required="true" Mask="false">808</Config>

```

Comment:
Is this a duplicate of "Web UI" above?

Reply:
I believe they are separate. The Web UI is the port to use and the Port for Web UI is a variable.